### ISSUE TITLE
Make an "incomplete" version of coins

### ISSUE THREAD

**Issue by bangetto** [IThundxr/CreateNumismatics#82]
### Describe the Suggestion

Add incomplete versions of coins that could be used in datapacks as a transitional item in a for example sequenced assembly

### Additional Context

currently there's no good way to do this if you don't make a mod

**Comment by brettgoodrich** [comment]
I believe a foundational feature of the currency is that it is uncraftable, thus leaving the control of its availability entirely in the hands of server admins.

**Comment by Opoodoop** [comment]
> I believe a foundational feature of the currency is that it is uncraftable, thus leaving the control of its availability entirely in the hands of server admins.

I disagree. Having an intermediary item would make it easier to develop mods/datapacks that add methods of obtaining like crafting recipes. It would also bring the options closer together in a way that matches the numismatics mod more cohesively.

**Comment by K2568** [comment]
I agree with opopdoop, if theres an intermediate item you can make sequenced assembley recipes for coins with only a datapack and no need for kubejs or a dedicated mod.

**Comment by Opoodoop** [comment]
For those still interested in the features discussed in this thread (i know i was), i made a quick mockup of these incomplete coins as a separate neoforge mod available here: [github](https://github.com/Opoodoop/numismatic_incomplete) and [modrinth](https://modrinth.com/mod/numismatics-incomplete)
Happy to have someone with more knowledge of modding help expand support to all versions and loaders that Numismatics supports.
I hope you'll forgive my mod quality for it is my first ever mod!

**Comment by qw1e2q34rr4funny-sys** [comment]
One problem i see with being able to craft coins is that, with the right setup, you can generate infinite money and crash the economy of the server your playing on. Though this can be simply resolved by not making the recipe renewable.

**Comment by Opoodoop** [comment]
> One problem i see with being able to craft coins is that, with the right setup, you can generate infinite money and crash the economy of the server your playing on. Though this can be simply resolved by not making the recipe renewable.

While yes that is true. I think it does not reduce the need for enabling modpack developers or server administrators to easily be able to customize the gameplay experience to fit their theme. I think this should be up to them.

**Comment by qw1e2q34rr4funny-sys** [comment]
I understand that now, thank you.
